spec/Service/StoreApiConnectorSpec.php

```diff
@@ -50,7 +50,7 @@ public function let(
         \putenv('SHOPWARE_SHOP_DOMAIN=');
     }
 
-    public function it_can_load_a_plugin_with_correct_credentials(
+    public function it_can_load_a_plugin_if_domain_only_exist_in_partner_account(
```

Comment:
Suggestion: Better english would be:
`it_can_load_a_plugin_only_if_domain_exists_in_partner_account`

Reply:
Done.